.github/workflows/test.yml

```diff
@@ -130,31 +128,21 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Cache pip
-      uses: actions/cache@v2
-      with:
-        # This path is specific to Ubuntu
-        path: ~/.cache/pip
-        # Look to see if there is a cache hit for the corresponding requirements file
-        key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt', 'doc/requirements.txt') }}
-        restore-keys: |
-          ${{ runner.os }}-pip-
-          ${{ runner.os }}-
```

Comment:
Is it OK to stop caching the packages? Or is it no longer reasonably possible to store the cache since the packages are now partially controlled via the Noxfile?

Reply:
I don't know how the cash works. The packages are installed by the noxfile. Maybe we can use `hashFile('noxfile.py')` for the cache.